src/styles/_colors.scss

```diff
@@ -1,9 +1,12 @@
 // NOTE: DON'T USE THIS FILE WHILE index.css is not being replaced with _index.scss
 
-@mixin colors {
-  // Font colors
-  --body-text-color: black;
+// @mixin colors {
```

Comment:
Prefer either removing it or leaving it, I don't see the point of commenting out unused mixins.

Reply:
True, trash is trash, I'll remove them now.